pkg/generator/helper.go

```diff
@@ -15,9 +15,7 @@ limitations under the License.
 */
 package generator
 
-import (
-	"html/template"
-)
+import "html/template"
```

Comment:
afaik, html compared to text just add some escaping function, avoid some malicious injection, such as xss attack, etc.. Here we don't need to load the generated code into the server, using text should be enough. cc @LinuxSuRen

Reply:
+1 from me